proposals/lip-0063.md

```diff
@@ -565,25 +565,25 @@ Once the snapshot block is final, a patched version of Lisk Core v4 will be rele
-
 ### Bootstrap Period
 
-There will be a bootstrap period in which a fixed set of delegates will constitute the set of active delegates. This period will last for one week assuming no missed blocks. This is done by setting the [`initRounds`](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-assets-schema) property in the snapshot block to the corresponding value, which is given by `DPOS_INIT_ROUNDS`. The reason to have this period is that the DPoS module selects delegates only with a valid BLS key outside of the bootstrap period. Thus, without a bootstrap period, the DPoS module could not select any delegates right after the hard fork which would result in a stop of the chain.
+There will be a bootstrap period in which a fixed set of delegates will constitute the set of active delegates. This period will last for one week assuming no missed blocks. This is done by setting the [`initRounds`](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0057.md#genesis-assets-schema) property in the snapshot block to the corresponding value, which is given by `DPOS_INIT_ROUNDS`. The reason to have this period is to achieve that the DPoS module selects only delegates with a valid BLS key. And this is realized by banning every delegate in the snapshot block. The unbanning is triggered by a _[register keys](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0050.md#register-keys)_ transaction. This way, delegates are forced to submit a _register keys_ transaction, and therefore to register a valid BLS key and a (new) generator key, in order to become an active delegate.
```

Comment:
Maybe this change is still upcoming, but where do we call the function `unbanDelegate`? Or where are we planning to call it?